apps/openmw/mwrender/sky.hpp

```diff
@@ -9,6 +9,8 @@
 #include <osg/Vec4f>
 #include <osg/Uniform>
 
+#include <osgParticle/BoxPlacer>
```

Comment:
It shouldn't be necessary to include it here exactly, just forward-declare BoxPlacer and revert dependency changes in sky.cpp.